bindata/manifests/metrics-exporter/metrics-service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: sriov-network-metrics-exporter-service
+  namespace: {{.Namespace}}
+  annotations:
+    prometheus.io/target: "true"
+spec:
+  selector:
+    app.kubernetes.io/name: sriov-metrics-exporter
+  ports:
+    - protocol: TCP
+      port: 9808
+      targetPort: 9808
```

Comment:
The POD is using hostNetwork. The targetPort is quite common and may conflict with other services on the host. Do we want to make exporter port configurable?

Reply:
Good point: I made the listening port user adjustable. I also added a kube-rbac-proxy, as the metrics need to be exposed via https, at least in OpenShift